### DISCUSSION TITLE
[Feature request] Presets

### DISCUSSION THREAD

**Discussion by grzegor** [anonaddy/anonaddy#372] [Feature Requests]
During an alias creation a user would have a drop down menu with all their presets, for example
```
Work
Private
Forums
Custom
```
and for example my `Work` would be
```
alias domain: myCustomWorkDomain.tld
alias format: random words
recipients: myRealWorkAddres@work.tld
```
and my `Forums` preset
```
alias domain: myPrivDomain.tld
alias format: UUID
recipients: myRealPrivAddress.tld
```
and `Custom` being the current behavior, when you have to fill in everything.

This would allow that future additions to the service (like #302 and #303 I posted today ;) ) would not clutter the UI and force users to decide a milion things while creating.

**Comment by willbrowningme** [comment]
That's not a bad idea, it may be a little complicated to implement though.
